front/pages/api/stripe/webhook.ts

```diff
@@ -601,14 +588,20 @@ async function handler(
                 status: "ended",
                 endDate: new Date(),
               });
-              const workspaceId = matchingSubscription.workspace.sId;
-              const auth = await Authenticator.internalAdminForWorkspace(
-                workspaceId
-              );
-              await revokeUsersForDowngrade(auth);
```

Comment:
I assume users won't be able to access / act on the workspace in any way after this, so fine to not revoke users, and remove checkStaticDatasources & agent archiving
However, should we keep the connections live for 15 days? might be a bit costly, so we could delete directly? (since it's not a real data loss) or maybe at least stop the syncs?

Reply:
actually, not very strongly held, especially if this requires additional logic for users who resuscribe before 15 days
feel free to ignore if you think it's not an issue

Reply:
That's what was decided in the design issue. Can revisit later indeed if it ends up being too expensive